pytrip/lib/cntr.c

```diff
@@ -0,0 +1,2038 @@
+/* -*- mode: c; c-basic-offset: 4 -*- */
+
+/*
+  cntr.c
+  General purpose contour tracer for quadrilateral meshes.
+  Handles single level contours, or region between a pair of levels.
+
+    The routines that do all the work, as well as the explanatory
+    comments, came from gcntr.c, part of the GIST package. The
+    original mpl interface was also based on GIST.  The present
+    interface uses parts of the original, but places them in
+    the entirely different framework of a Python type.  It
+    was written by following the Python "Extending and Embedding"
+    tutorial.
+
```

Comment:
Consider adding a link where it is from?

Reply:
added